rcsbapi/graphql_schema.py

```diff
@@ -1072,23 +1083,42 @@ def add_field_args(self, field_name: str, args: List[dict[str, Any]], query_args
         """Add arguments to a field, returning the fieldname and args as a formatted string.
 
         Args:
-            args (list[dict[str, Any]]): args of a field, retrieved from the GraphQL schema/FieldNode object
-            query_args (dict[str, Any]): dictionary where keys are argument name and values are user input
+            field_name (str): name of the field
+            args (list[dict[str, Any]]): argument definitions from schema
+            query_args (dict[str, Any]): full dictionary of query arguments
 
         Returns:
-            str: field name or field name with corresponding arguments
+            str: field name, potentially with arguments applied (e.g., field(arg: val))
+
+        Raises:
+            ValueError: if an invalid field argument key is found
         """
-        # Check FieldNode argument names and see if user has passed in corresponding values
+        field_args_dict = query_args.get("datalistargs", {})
```

Comment:
So this `datalistargs` is originally `data_list_args`? That means we are still stripping underscores then? I thought since fixing the `from_` issue, we can avoid stripping underscores. Can that be done?